### PR TITLE
Adds PageInfo support to QueryExpression. 

### PR DESCRIPTION
**Pull request by jheth**
@mbhnyc  I saw you requested `PageInfo` support in #55.  I took a shot at it.  What do you think?

### CODE REVIEW

File: spec/lib/xml/query_expression_spec.rb [hunk 1]
```diff
@@ -36,4 +36,29 @@
       it { expect(subject.to_xml).to include("<b:FilterOperator>And</b:FilterOperator>").or(include("<b:FilterOperator>Or</b:FilterOperator>")) }
     end
   end
+
+  describe 'PageInfo' do
+    subject {
+      query = DynamicsCRM::XML::QueryExpression.new('account')
+      query.columns = %w(accountid name)
+      query.criteria.add_condition_expression('name', 'NotEqual', 'Test Account')
+      query.page_info = DynamicsCRM::XML::PageInfo.new(count: 5, page_number: 2, return_total_record_count: true)
```
Comment (jheth) [inline spec/lib/xml/query_expression_spec.rb]: @mbhnyc  Here's how it can be used with a `QueryExpression`

File: lib/dynamics_crm/client.rb [hunk 2]
```diff
@@ -141,7 +141,13 @@ def rollup(target_entity, query, rollup_type="Related")
         })
     end
 
-    def retrieve_multiple(entity_name, criteria=[], columns=[], operator=nil)
+    def retrieve_multiple_query(query_expression)
```
Comment (mbhnyc) [inline lib/dynamics_crm/client.rb]: also a few lines below this is an unnecessary `return`

Comment (jheth) [inline lib/dynamics_crm/client.rb]: I don't want to keep adding parameters to `retrieve_multiple`, they aren't named parameters and you have to remember the order.  I like building a `QueryExpression` by itself and then passing that in as a single parameter.

Comment (jheth) [inline lib/dynamics_crm/client.rb]: I may have `retrieve_multiple` support both.  I would check the first parameter (`entity_name`) to see if it is an `instance_of?(QueryExpression)` and use that, then we wouldn't need the separate method.

**Comment by jheth** [comment]
@mbhnyc I've figured it out.  See the updated README for the example.

**Comment by jheth** [comment]
resolves #48

**Pull request merged**